UI/Web/src/app/series-detail/series-detail.component.ts

```diff
@@ -247,16 +247,27 @@ export class SeriesDetailComponent implements OnInit, OnDestroy {
 	onScroll(): void {
 		const tabs = document.querySelector('.nav-tabs') as HTMLElement | null;
 		const main = document.querySelector('.main-container') as HTMLElement | null;
-		const content = document.querySelector('.tab-content') as HTMLElement | null;
-		let mainOffset = main!.offsetTop;
-		let tabOffset = tabs!.offsetTop;
-		let contentOffset = content!.offsetTop;
-		let mainScrollPos = main!.scrollTop;
+    const info = document.querySelector('.info-container') as HTMLElement | null;
+		const mainOffset = main!.offsetTop;
+		const mainScrollPos = main!.scrollTop;
+    const tabsWidth = tabs!.clientWidth;
```

Comment:
Move these variables closer to where they are used, like we only need tabsWidth in the if statement.

Reply:
Resolved in https://github.com/Kareadita/Kavita/pull/1322/commits/f7ec0a67b1d0c0d34b5da56f8345cfd9d9614817